src/ApiCompat/Rules/Compat/DifferenceRule.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Composition;
+
+namespace Microsoft.Cci.Differs.Rules
+{
+    internal abstract class DifferenceRule : Microsoft.Cci.Differs.DifferenceRule
```

Comment:
LOL. Too scared to update the base rule and just make it optional?

Reply:
I don't really like playing games and using the same type name in a different namespace as it just causes confusion. I'm OK with it staying in ApiCompat but can you change the name of the type to "CompatDifferenceRule" or something along those lines (yes I realize you have to then update all the base types of the rules but you are already touching those files anyway).

Reply:
Ok, I had done that first, but you know how much I hate naming...  I'll use CompatDifferenceRule.